### PR TITLE
Upgrade to JSX 3

### PR DESCRIPTION
**Pull request by EnriqueVidal**
I switched to JSX 3 and since no test suite is in place also upgraded to the latest bs-platform to make sure build happens.

Rel: #12

### CODE REVIEW

**Review by JasoonS — APPROVED** [review]
Seems good to me :)

**Comment by yawaramin** [comment]
@EnriqueVidal if this library doesn't use JSX, can you remove the `react-jsx` config from the `bsconfig.json`.

File: bsconfig.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "bs-date-fns",
   "bsc-flags": ["-bs-super-errors"],
-  "reason": {"react-jsx": 2},
+  "reason": {"react-jsx": 3},
```
Comment (Yakimych) [inline bsconfig.json]: Hi! Can we remove this line altogether? Doesn't look like React/JSX is used anywhere in this library, but it does mess with bs-platform 8.2.0.

Comment (EnriqueVidal) [reply bsconfig.json]: Sure, done.

**Comment by romanschejbal** [comment]
When can we expect this to be merged? :-)

**Comment by froatsnook** [comment]
Thanks for the merge 🥳

**Pull request merged**